docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,28 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Rename tag
+
+*MSS*
+
+. User requests to rename the tag "friends" to "buddies"
```

Comment:
The format of the use case is perfect! It shows a clear interaction between the user and the address book. 👍 
Since specific tags are used, the use case itself becomes of limited scope. You can eliminate specific tags to make it generic to the scenario.